regression/barebones/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/hyperledger/fabric-test/regression/barebones
+
+go 1.13
+
+require (
+	github.com/hyperledger/fabric-test v1.4.4
```

Comment:
shouldn't this point to fabric-test master?

Reply:
Corrected, I forgot to add the replace directive